### ISSUE TITLE
Add --chrome-flags to CLI

### ISSUE THREAD

**Issue by paulirish** [GoogleChrome/lighthouse#1443]
Usecase: 

```sh
lighthouse --chrome-flags="--ignore-certificate-errors" https://localbuild.dev/site/
```

(Broken out from #1416)

**Comment by brendankenny** [comment]
and (from other discussions), preferably we'd have some way of also passing in the sub-sub-flags to v8 via `--js-flags="--trace_deopt --etc"`.

Not sure if it should be a separate `--js-flags` flag or if it should be nested inside `--chrome-flags` so the value could just be passed to chrome directly (nested quotes might get weird, though).

**Comment by paulirish** [comment]
sgtm!  I'd prefer we try the nested quotes route first, otherwise lighthouse will have to eat some complexity.

**Comment by paulirish** [comment]
The current alternative to this is running the `chrome-debug` binary, which lighthouse adds when you do a global install. It can take additional flags..

```sh
chrome-debug --show-paint-rects
```

then run lighthouse, however you like.

Lighthouse will automatically use the chrome-debug instance. as it's got port 9222 open to connect to.

**Comment by paulirish** [comment]
fixed by #1761
